tests/common/hypothesis-chrome-extension-test.js

```diff
@@ -111,6 +109,7 @@ describe('HypothesisChromeExtension', function() {
       removeFromTab: sandbox.stub().returns(Promise.resolve()),
     };
     fakeErrors = {
+      AlreadyInjectedError: function AlreadyInjectedError() {},
       shouldIgnoreInjectionError: function() {
```

Comment:
This mock was missing. It happened to work because proxyquire "calls through" to the original module for exports that are missing in the mock.